app/models/drawable.ts

```diff
@@ -0,0 +1,51 @@
+// File: graph.ts
```

Comment:
Not really important, but this should probably be updated. Also adding a `Contributor` field would be nice, but not necessary.